at_secondary/at_secondary_server/lib/src/server/bootstrapper.dart

```diff
@@ -48,7 +51,13 @@ class SecondaryServerBootStrapper {
       secondaryServerInstance.setExecutor(DefaultVerbExecutor());
       secondaryServerInstance
           .setVerbHandlerManager(DefaultVerbHandlerManager());
-      await secondaryServerInstance.start();
+
+      //starting secondary in a zone
+      runZonedGuarded(() async {
+        await secondaryServerInstance.start();
+      }, (error, stackTrace) {
+        logger.severe('Uncaught error: $error');
```

Comment:
@srieteja Can you do another PR please which also logs the stack trace?

Reply:
@gkc I could add that additional log to this PR itself. Or should I do a new PR entirely?

Reply:
Oh yes, please add in this PR. Sorry I thought this PR had been closed already. Thanks @srieteja !